Server/validation/joi.js

```diff
@@ -43,6 +43,10 @@ const newPasswordValidation = joi.object({
   confirm: joi.string(),
 });
 
+const userDeleteValidation = joi.object({
```

Comment:
Can you please rename this `deleteUserParamValidation` to be consistent with the other validation naming schemes?